src/pages/billing/Project.js

```diff
@@ -58,7 +114,9 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
     setDeletingUser(false)
     setUpdating(false)
     setEditingUser(false)
-
+    const rawWorkspaces = await Ajax(signal).Workspaces.list()
+    console.log(rawWorkspaces)
+    setWorkspaces(_.map(workspaceResponse => workspaceResponse.workspace, _.filter(workspaceResponse => workspaceResponse.workspace.namespace === projectName, rawWorkspaces)))
```

Comment:
This is a personal preference, so please feel free to ignore, but I think the variable name `workspaceResponse` here is long-enough that it takes up more space than it deserves considering its function as just a way to access nested data. I would shorten it to `response` or `wr` or even `x`. I use `x` when (a) the context of its use is extremely narrow as it is in this case and (b) to communicate that the name is really, really unimportant. In this case, it may be quite clear from locality that `rawWorkspaces` is a workspace response object, so calling that out is less helpful to the reader than highlighting (via percentage of space taken) what is happening with it.

Reply:
In general, CI has had the practice of verbose variable names for understanding, over conciseness. However, I think `response` is just as clear as `workspaceResponse`, and cuts down the verbosity considerably, so I think that is a clear win. Will do.